corehq/apps/hqadmin/templates/hqadmin/admin_restore.html

```diff
@@ -64,9 +67,11 @@
                                         {% endfor %}
                                     </tbody>
                                 </table>
+                    {% if not hide_xml %}
                             </div>
                         </div>
                     </div>
+                    {% endif %}
```

Comment:
This is kinda weird.  Could you instead just hide the xml part (and replace it with "xml hidden" or something), rather than hiding the wrapper stuff?  Just thinking that this will make it hard to modify the html of this page in the future.

Reply:
good call, updated